pkg/operator/ceph/config/datapath.go

```diff
@@ -39,6 +39,15 @@ type DataPathMap struct {
 	// persisted to the host. The log dir is always /var/log/ceph. If logs are not persisted to the
 	// host, logs are not shared between containers via empty dir or any other mechanism.
 	HostLogDir string
+
+	// ContainerLogDir represents Ceph's logging directory
+	ContainerLogDir string
+
+	// HostCrashDir represents Ceph's crash reports directory on the host
+	HostCrashDir string
+
+	// ContainerCrashDir represents Ceph's crash reports directory
+	ContainerCrashDir string
```

Comment:
The container crash dir is constant in the container. I would  suggest removing this so that it doesn't invite misuse in the future.

Ditto the constant function option here as well.
```go
func (d *DataPathMap) ContainerCrashDir() string {
	return config.VarCrashCephDir
}